### PR TITLE
Add rule to main.scss - disabled form fields text to stylistically match normal fields

### PR DESCRIPTION
**Pull request by CodeKrakken**

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -262,4 +262,17 @@ formly-array-type {
...
 }
 
+.mdc-text-field--disabled, 
+.mdc-text-field--disabled .mat-mdc-input-element {
+  color: black !important;
+}
+
+.mdc-text-field--disabled .mdc-floating-label {
+  color: rgba(0, 0, 0, 0.6)
+}
```
Comment (lukestanley) [inline src/styles/main.scss:272]: What's the purpose of this? @CodeKrakken

Comment (CodeKrakken) [reply src/styles/main.scss:272]: @lukestanley For consistency of style with the labels of regular input fields - they are not black but slightly grey.

File: package-lock.json
Comment (lukestanley) [inline package-lock.json:1]: Please commit changes to files that are unrelated to the code you are working on. This is more easily avoided by not avoiding automatic staging of changed files! @CodeKrakken

**Comment by lukestanley** [comment]
To test I made this flow:
/workflow-builder?data=NobwRALgngDgpmAXGAZgewE4FswBowBWAzmgHYDKAxgBZxYCGS408SYaARgXJRHmDAxp4GCAEs4RJgKEjoARmksEyIhAxjSAc37iIAGxVgACrLiioAAkUBffIOHnoAJiWwjajdt1iDR044Wlq74GHD0ACYA8qT6UEjqAK5wNqn4iWJUtAxMdmAc+miUANYAwmhYWHCkfMhgNgC6QA
(append the path to a build to have the flow loaded).

File: package-lock.json
Comment (CodeKrakken) [reply package-lock.json:1]: @lukestanley I've reverted the package lock - that should be fine now!

**Comment by lukestanley** [comment]
@CodeKrakken I thought we had merged this in already but we haven't, can you recall any particular reason why (other than us being busy and distracted)?

**Comment by CodeKrakken** [comment]
@lukestanley Nothing standing in the way of a merge as far as I can see.

**Comment by lukestanley** [comment]
@CodeKrakken I don't think this was marked approved by someone after charges had been requested. In the future please do not merge something unless it has been marked approved!
